Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -83,9 +84,20 @@ AzAssetBrowserWindow::AzAssetBrowserWindow(QWidget* parent)
     m_ui->m_assetBrowserTableViewWidget->setVisible(false);
     m_ui->m_toggleDisplayViewBtn->setVisible(false);
     m_ui->m_searchWidget->SetFilterInputInterval(AZStd::chrono::milliseconds(250));
+
+    m_ui->m_collapseAllButton->setAutoRaise(true); // hover highlight
+    m_ui->m_collapseAllButton->setIcon(QIcon(":/stylesheet/img/24x24/layers.png"));
+    connect(
+        m_ui->m_collapseAllButton, &QToolButton::released, this,
```

Comment:
This should probably be clicked instead of released so all of the button presses are handled consistently. 
Is there a context menu or shortcut key to collapse all?

Reply:
Just changed it from released to pressed. There is no context menu or shortcut key but I think that for the collapse all behavior is not necessary.